build-system/global-configs/prod-config.json

```diff
@@ -28,6 +28,7 @@
   "amp-auto-ads-adsense-holdout": 0.1,
   "amp-auto-lightbox": 1,
   "amp-date-picker": 1,
+  "amp-force-prerender-visible-elements": 0.01,
```

Comment:
We need to keep the percentage the same in both configs, that way we can see the impact in the canary/rc graph. So probably, 1% of canary and 1% of prod traffic.

Reply:
Done